### PR TITLE
feature: enable security by default

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

**Comment by sbender9** [comment]
@tkurki I'm currently just enabling by default for new installs. Not sure if we should eventually enable for existing installs also?

**Comment by tkurki** [comment]
`strategy` should get created also when ~/.signalk is created spontaneously, without running the setup script.

How about adding instructions to the admin ui login screen on how to disable security in case you have misplaced your login credentials?

**Comment by sbender9** [comment]
I think merge #430 first, then I can fix this for when ~/.signalk is created spontaneously

Personally, I don't like the idea of providing instructions to disable security on the login page. Just seems  like wrong thing to do. How about eventually adding it to the README so that it's easily googleable?

**Comment by tkurki** [comment]
I was thinking about something in the failed login message, maybe after several failed login attempts. Google does not help if you are offline.

But I'm good without it as well.

File: lib/serverroutes.js [hunk 1]
```diff
@@ -345,9 +345,8 @@ module.exports = function(app, saveSecurityConfig, getSecurityConfig) {
       securityWasEnabled = true
 
       function addUser(req, res, securityStrategy, config) {
-        var config
         if (!config) {
-          config = getSecurityConfig(app)
+          config = app.securityStrategy.getConfiguration()
```
Comment (tkurki) [inline lib/serverroutes.js:349]: I always consider function parameters immutable, so I preferred the original in that respect. But that was overloading the name.

**Pull request merged**